Dmitriy_Vysotskyy2.0/Steps/AddToCartSteps.cs

```diff
@@ -0,0 +1,44 @@
+using Dmitriy_Vysotskyy2._0.Builders;
+using Dmitriy_Vysotskyy2._0.Common;
+using FluentAssertions;
+using TechTalk.SpecFlow;
+
+namespace Dmitriy_Vysotskyy2._0.Steps;
+
+[Binding]
+public class AddToCartSteps : FeatureHelper
+{
+    private string _alertText;
+    
+    [Given(@"a user has logged in")]
```

Comment:
Why do you need to duplicate this step in many features? Why don't you create step in new class and add multiple annotations to it? 

![image](https://github.com/dimavys/Dmitriy_Vysotskyy2.0/assets/47978115/df0a9a59-2c5d-43fa-b2ce-19ff5c7f1d38)